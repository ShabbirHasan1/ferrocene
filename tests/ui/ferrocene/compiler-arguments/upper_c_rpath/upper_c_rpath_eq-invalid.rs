// check-fail
// compile-flags: -Crpath=invalid

fn main() {}

// ferrocene-annotations: um_rustc_C_rpath
