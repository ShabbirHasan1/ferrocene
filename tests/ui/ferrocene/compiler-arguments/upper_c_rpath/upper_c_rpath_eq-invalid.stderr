error: incorrect value `invalid` for codegen option `rpath` - one of: `y`, `yes`, `on`, `n`, `no`, or `off` was expected

